src/components/TabMenu/Item/Item.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import styles from './Item.module.css';
+
+export interface ItemProps extends React.HTMLAttributes<HTMLLIElement> {
+  active?: boolean;
+}
+
+const Item: React.FC<ItemProps> = ({
+  className: customClassName,
+  active = false,
+  children,
+  onClick = () => {},
+  ...props
+}) => {
+  const isChildActive = (child: React.ReactNode) => {
+    if (React.isValidElement(child)) {
+      const activeChild = child.props?.className?.indexOf('active') >= 0;
+      if (activeChild) {
+        console.warn(
```

Comment:
This seemed like a reasonable way to add deprecations for now since we don't have many.

Reply:
Good call, but it still adds tech debt. How would you feel about looking for instances that use this, squash them, and eliminate the code?

Or just making an issue to track that work.

Reply:
On second thought, this may specifically be to cover react router's `Link` components.
If that's the case, this might be a helpful, although slightly weird, feature.

Reply:
That is actually a very good point. I'll take a closer look at that possibility on Monday. Thanks for catching this!

Reply:
I think the only thing this PR is waiting for is the answer to this question.
If we find out it's a NavLink / Link enabler, then the only code change would be switching from a `console.warn` to a comment explaining why the weird code is here.

Reply:
@mdespuits did you ever look into this? And can you or @pixelbandito explain what's happening here so I can try to resolve the issue?